airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/state/object_storage/ObjectStorageDestinationStateManager.kt

```diff
@@ -126,18 +133,47 @@ class ObjectStorageDestinationState(
     }
 
     /** Used to guarantee the uniqueness of a key */
```

Comment:
Add comment describing how we ensure uniqueness by looking up the ordinal postfix by searching s3 and incrementing here.